spring-cloud-task-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -217,24 +132,23 @@ default, it imports an additional configuration class (`SimpleTaskConfiguration`
 additional configuration registers the `TaskRepository` and the infrastructure for its
 use.
 
-Out of the box, the `TaskRepository` uses an in-memory `Map` to record the results
-of a task. A `Map` is not a practical solution for a production environment, since
-the `Map` goes away once the task ends. However, for a quick getting-started
-experience, we use this as a default as well as echoing to the logs what is being updated
+In our demo, the `TaskRepository` uses an embedded H2 database to record the results
+of a task. This H2 embedded database is not a practical solution for a production environment, since
+the H2 DB goes away once the task ends. However, for a quick getting-started
```

Comment:
Remove the period after "DB"

Reply:
hmm... What period after "DB"?

Reply:
OK. Remove the speck of dirt on my screen after "DB". ;)